grid_map_sdf/package.xml

```diff
@@ -16,12 +16,12 @@
   <build_depend>grid_map_cmake_helpers</build_depend>
 
   <depend>grid_map_core</depend>
-  <!-- <depend>pcl_ros</depend> -->
+  <depend>libpcl-all-dev</depend>
```

Comment:
Is there a specific submodule(s) of PCL this is using? It would be good to specify just the ones you need rather than installing ALL of PCL including things like visualizers and debuggers

Reply:
I'm not aware of a rosdep key available to install part of pcl. (we could still make it to `libpcl-all` instead of `dev` though)
https://github.com/ros/rosdistro/blob/master/rosdep/base.yaml#L4093

This installs all pcl, but I think only required module (common module) is linked since it is specified with the following lines in the CMakeLists.txt.
`find_package(PCL REQUIRED COMPONENTS common)`

Reply:
FYI, `pcl_ros` used to have [dependency to `libpcl-all-dev`](https://github.com/ros-perception/perception_pcl/blob/6900f7cf650e6c0df2aef45e0011833905b0ba9e/pcl_ros/package.xml#L38) in ROS1, so it's very likely to have the same dependency in ROS2. Therefore, I would say this PR isn't increasing the number of installed packages compared to ROS1 implementation.